src/polyhedral/butterfly.js

```diff
@@ -25,6 +25,7 @@ export default function(faceProjection) {
             : lambda < pi / 2 ? phi < 0 ? 3 : 1
             : phi < 0 ? 7 : 5];
       })
-      .scale(101.858)
-      .center([0, 45]);
+    .angle(-30)
+    .scale(101.858)
+    .center([0, 45]);
```

Comment:
These should all be four-space indents because it is method chaining on the same object. (This comment applies to multiple files in this PR.)